src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Aead.cs

```diff
@@ -86,6 +86,81 @@ internal static partial class AppleCrypto
                 }
             }
         }
+        internal static unsafe void AesGcmEncrypt(
```

Comment:
It looks like there's not a blank line between this method and the previous one?

```suggestion

        internal static unsafe void AesGcmEncrypt(
```